resources/default_pipeline_environment.yml

```diff
@@ -628,6 +626,7 @@ steps:
     deployIdLogPattern: '^.*"xs bg-deploy -i (.*) -a .*".*$'
     loginOpts: ''
     deployOpts: ''
+    dockerPullImage: false
```

Comment:
Now that the general default is true, we need to duplicate the xsDeploy step default (`false`) as non-nested as well. Previously it was only necessary to set the step default nested inside `docker` (line 632)